src/app/shared/conditional-show/conditional-show.directive.ts

```diff
@@ -100,6 +100,13 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
 
   private show() {
     this.el.nativeElement.hidden = false;
+    if (this.caseField && this.caseField.field_type && this.caseField.field_type.type !== 'Collection') {
```

Comment:
is `this.caseField && this.caseField.field_type` needed? 
if this.caseField was null, other parts of this class would fail. Also, the property `field_type` is mandatory. 
I'm not 100% but can you try to remove `this.caseField && this.caseField.field_type` and see if it still works?

Reply:
field_type is mandatory, but in it is failing most of the test case mocks so to be on safer side, I have added this check.